app/controllers/account/preferences_controller.rb

```diff
@@ -100,7 +100,7 @@ def update_pref(pref, val)
     def update_content_filter(pref, val)
       filter = ContentFilter.find(pref)
       @user.content_filter[pref] =
-        if filter.type == :boolean && filter.prefs_vals.count == 1
+        if filter.type == :boolean && filter.prefs_vals.length == 1
```

Comment:
Though it's not the point of this PR, can you use
`one?`
instead of 
`length == 1`

Reply:
Yes! I don't know that method. Will alter